### ISSUE TITLE
[BUG] Lower buttons on /speak ("skip", "submit",...) aren't visible on smaller screens

### ISSUE THREAD

**Issue by stefangrotz** [common-voice/common-voice#3969]
**Describe the bug**
Lower buttons on /speak ("skip", "submit",...) aren't visible on smaller screens

**To Reproduce**
1. Go to https://commonvoice.mozilla.org/en/speak using a laptop with 1028p resolution
3. Don't see the buttons
4. Zoom out
5. See the buttons

**Expected behavior**
It should be responsive

**Screenshots**
![image](https://user-images.githubusercontent.com/5982239/224284936-2a79a14c-2971-4b31-bd1c-98b52cd9c48c.png)

I used Chrome on Windows 10.

**Comment by 2subham5** [comment]
I want to work on this issue . I think I can resolve this .
Plz do assign me #3969.

**Comment by 2subham5** [comment]
Where is the html file present for the footer.

**Comment by shashankcoc** [comment]
Is this issue still open @stefangrotz ?

**Comment by stefangrotz** [comment]
Yes, it is still an issue for me. My OS scales everything up to 125%, so this might help to reproduce it.

**Comment by moz-bozden** [comment]
This issue is related to the fact the designs are based on 1080p screens, which is the most used screen size globally (at that time).
So with 1028p you get problems.
The related CSS also fixes these, and they are not "fluid" i.e. responsive...
This will need a larger redesign on all pages.
